flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/source/TestMetadataTableReadableMetrics.java

```diff
@@ -219,27 +219,27 @@ public void testPrimitiveColumns() throws Exception {
 
     Row binaryCol =
         Row.of(
-            59L,
```

Comment:
The column sizes have changed cause the new writer picks up all table properties, which was not true before. I validated the actual values are correct using `parquet-tools`.

Reply:
I probably should have originally picked these up from the file itself, but its a suggestion for another PR

Reply:
Yep, we should do that eventually.